From 152a0a5dd7bc9ecc0d4adffd077d0368b021f3d2 Mon Sep 17 00:00:00 2001
From: Robert Lai <c_rlai@codeaurora.org>
Date: Mon, 14 Sep 2015 10:20:56 -0700
Subject: [PATCH] mm-video: vidc: common: Fix debug messages

Change logging level to reduce excessive logging during
normal execution.

Change-Id: I29b4c0f9422ea26067015e2b5758a0f47b0d1260
---
 libc2dcolorconvert/C2DColorConverter.cpp               | 4 ++--
 mm-video-v4l2/vidc/common/src/vidc_color_converter.cpp | 4 +++-
 2 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/libc2dcolorconvert/C2DColorConverter.cpp b/libc2dcolorconvert/C2DColorConverter.cpp
index 64ea771..8a319c0 100644
--- a/libc2dcolorconvert/C2DColorConverter.cpp
+++ b/libc2dcolorconvert/C2DColorConverter.cpp
@@ -561,7 +561,7 @@ size_t C2DColorConverter::calcLumaAlign(ColorConvertFormat format) {
         case NV12_128m:
           return 1;
         default:
-          ALOGE("unknown format passed for luma alignment number");
+          ALOGD("unknown format passed for luma alignment number");
           return 1;
     }
 }
@@ -576,7 +576,7 @@ size_t C2DColorConverter::calcSizeAlign(ColorConvertFormat format) {
         case NV12_128m:
           return ALIGN4K;
         default:
-          ALOGE("unknown format passed for size alignment number");
+          ALOGD("unknown format passed for size alignment number");
           return 1;
     }
 }
diff --git a/mm-video-v4l2/vidc/common/src/vidc_color_converter.cpp b/mm-video-v4l2/vidc/common/src/vidc_color_converter.cpp
index 413b27b..a80fb20 100644
--- a/mm-video-v4l2/vidc/common/src/vidc_color_converter.cpp
+++ b/mm-video-v4l2/vidc/common/src/vidc_color_converter.cpp
@@ -1,5 +1,5 @@
 /*--------------------------------------------------------------------------
-Copyright (c) 2012-2013, The Linux Foundation. All rights reserved.
+Copyright (c) 2012-2013,2015 The Linux Foundation. All rights reserved.
 
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions are
@@ -98,6 +98,8 @@ bool omx_c2d_conv::open(unsigned int height,unsigned int width,
     bool status = false;
 
     if (!c2dcc) {
+        DEBUG_PRINT_HIGH("C2D Open: Src: %d, Dst: %d, Width: %d, Height: %d",
+                src, dest, width, height);
         c2dcc = mConvertOpen(width, height, width, height,
                 src,dest,0,0);
 
-- 
1.9.1

